src/shared/conf-parser.c

```diff
@@ -419,11 +419,11 @@ int config_parse(
         if (ret_mtime)
                 *ret_mtime = mtime;
 
-        return 0;
+        return 1;
 }
 
 static int config_parse_many_files(
-                const char *conf_file,
+                const char* const* conf_files,
```

Comment:
not a fan of the "const" as mentioned, now people cannot pass in a regular strv without casting, only STRV_MAKE_CONST() works now